docs/src/pages/theming/responsive/react.mdx

```diff
@@ -123,3 +124,17 @@ When using the array syntax, specify each breakpoint styling in order from `base
     {' 🐕 '}
   </Text>
 </Example>
+
+## useBreakpointValue
+
+To achieve a similar effect for none style properties you can use the `useBreakpointValue` hook. The hook will accept the same object or array syntax as
```

Comment:
The wording I think is a bit weird, "To achieve a similar effect for none style properties"

Reply:
I agree this wording is weird, how about `A responsive design can be achieved for all properties using the useBreakpointValue hook.  The hook will accept the object or array syntax and return the value of the current breakpoint.` Also I will add another example showing the array syntax.

Reply:
LGTM